src/deepsparse/server/cli.py

```diff
@@ -206,6 +208,20 @@ def main(
        ...
     ```
     """
+
+    def _fetch_server(integration: str, config_path: str):
+        if integration == "local":
+            return DeepsparseServer(server_config=config_path)
+        elif integration == "sagemaker":
+            return SagemakerServer(server_config=config_path)
+        elif integration == "openai":
+            return OpenAIServer(server_config=config_path)
+        else:
+            raise ValueError(
+                f"{integration} is not a supported integration. Must be "
+                "one of local, sagemkaer or openai."
+            )
```

Comment:
nit: typo, also maybe worth defining a list of supported integrations as a global constant? Then we can just print out the list of supported integrations instead of having this as a hardcoded comment